run.py

```diff
@@ -81,5 +120,7 @@ def profile_one_step(func, nwarmup=3):
 
     if args.profile:
         profile_one_step(test)
+    elif args.cudastreams:
+        run_one_step_with_cudastreams(test, 10)
```

Comment:
May I ask why the number of streams is set to 10? Is it better to specify the number of streams through the argument `--cudastreams`?

Reply:
> May I ask why the number of streams is set to 10? Is it better to specify the number of streams through the argument `--cudastreams`?

I thought an arg would make it more complicated and I don't think there's an actual benefit. If you can't learn from streams <= 10 then you're barking up the wrong tree. The few times I ran it with high stream count I wanted an iteration step, etc. And still it didn't show anything.